profiles/minim.yml

```diff
@@ -20,7 +20,8 @@ packages:
   - kmod-nf-nathelper
   - kmod-nf-nathelper-extra
   - minim-web-admin
-  - px5g-standalone
+  #- px5g-standalone
```

Comment:
no need to leave the commented out px5g-standalone

Reply:
As per the discussion I am removing the px5g-standalone package and adding the openssl-util package.

Reply:
Right, but you don't need to have a line with a commented out px5g-standalone.  Remove it completely

Reply:
Ok got it.

Reply:
@bmastbergen Done.